docs/content/users/quickstart.md

```diff
@@ -375,6 +375,44 @@ While the generic `php` project type is [ready to go](./project.md) with any CMS
 
     For more advanced tasks like adding elasticsearch, building and watching storefront and administration, see [susi.dev](https://susi.dev/ddev-shopware-6).
 
+=== "Silverstripe"
+
+    ## Silverstripe
+
+    === "Silverstripe"
```

Comment:
I would strike that single `silverstripe` tab. that way you have silverstripe three times in a row. first the active primary tab, then in the heading and then the label of the secondary tab. and then you have a tertiary tab row with composer and git clone. compared to the other quickstarters i would remove the single silverstripe tab for consistency reasons.

Reply:
Sorry, what do you mean?

Reply:
I've posted the comment before your latest commit that removes line 382 (`=== "Silverstripe"`) from the quickstarter.md around other things. looks good now, thanks for the fix!

Reply:
Yeah, I did "double-push", I left the first push because I was about to go to sleep, and intended to wrap up later the next day.

And then saw your comment, which confused me a bit, but makes sense you responded to my initial "fix" where I only fixed the indentation! Thanks for the clarification :)